include/hal/library/responder/setcertlib.h

```diff
@@ -38,6 +38,17 @@ extern bool libspdm_write_certificate_to_nvm(uint8_t slot_id, const void * cert_
                                              size_t cert_chain_size,
                                              uint32_t base_hash_algo, uint32_t base_asym_algo);
 
+/**
+ * Erase a certificate chain in non-volatile memory.
+ *
+ *
+ * @param[in]  slot_id          The number of slot for the certificate chain.
+ *
+ * @retval true   The certificate chain was successfully erased to non-volatile memory.
+ * @retval false  Unable to erase certificate chain to non-volatile memory.
+ **/
+extern bool libspdm_erase_nvm_certificate(uint8_t slot_id);
```

Comment:
we can reuse libspdm_write_certificate_to_nvm(slot_id, NULL, 0, 0, 0) as erase.

Reply:
Thanks. I reuse `libspdm_write_certificate_to_nvm(slot_id, NULL, 0, 0, 0)` as erase now.